test/database.ts

```diff
@@ -762,6 +782,83 @@ describe('Database', () => {
     });
   });
 
+  describe('blindWrite', () => {
+    const mutation = new Mutation();
+    mutation.insert('MyTable', {
+      Key: 'k3',
+      Thing: 'xyz',
+    });
+
+    const SESSION = new FakeSession();
+    const RESPONSE = {commitTimestamp: {seconds: 1, nanos: 0}};
+    const TRANSACTION = new FakeTransaction(
+      {} as google.spanner.v1.TransactionOptions.ReadWrite
+    );
+
+    let pool: FakeSessionPool;
+    let releaseStub;
+
+    beforeEach(() => {
+      pool = database.pool_;
+      (sandbox.stub(pool, 'getSession') as sinon.SinonStub).callsFake(
+        callback => {
+          callback(null, SESSION, TRANSACTION);
+        }
+      );
+    });
+
+    it('should return any errors getting a session', done => {
+      const fakeErr = new Error('err');
+
+      (pool.getSession as sinon.SinonStub).callsFake(callback =>
+        callback(fakeErr, null, null)
+      );
+
+      database.blindWrite(mutation, err => {
+        assert.deepStrictEqual(err, fakeErr);
+        done();
+      });
+    });
+
+    it('should return error when passing an empty mutation', done => {
+      const fakeError = new Error('err');
+      const fakeMutation = new Mutation();
+      try {
+        database.blindWrite(fakeMutation, (err, res) => {});
+      } catch (error) {
+        assert(error instanceof Error);
+      }
+      done();
+    });
+
+    it('should return error when passing null mutation', done => {
+      const fakeError = new Error('err');
```

Comment:
Use the actual error returned from backend

Reply:
Updated